wrappers/services.go

```diff
@@ -1789,7 +1789,10 @@ func generateOnCalendarSchedules(schedule []*timeutil.Schedule) []string {
 }
 
 type RestartServicesFlags struct {
+	// Reload set if we might need to reload the service definitions.
```

Comment:
If we always reload if this is set, then this might be clearer:

```suggestion
	// Reload set if we should reload the service definitions.
```